app/views/reservations/edit.html.haml

```diff
@@ -25,6 +25,13 @@
 
   = render "reservation_fields", f: f
 
+  .well
```

Comment:
I probably should have caught this sooner. We should match the style of the `new` form where this is above the times. 

Also, it should not _always_ appear. It is contingent on the user_notes_field_mode being either optional or required.

The `new` form uses this:

```haml
      - if show_note_input_to_user?(@order_detail)
        .span6
          = f.input :note,
            label: @order_detail.product.user_notes_label.presence,
            required: @order_detail.product.user_notes_field_mode.required?
```